src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -324,12 +325,19 @@ private function replacePropertyAccessor(ContainerBuilder $container): void
 
     private function configureTwigTextExtension(ContainerBuilder $container, XmlFileLoader $loader, array $config): void
     {
+        $bundles = $container->getParameter('kernel.bundles');
+
         $container->setParameter('sonata.admin.configuration.legacy_twig_text_extension', $config['options']['legacy_twig_text_extension']);
         $loader->load('twig_string.xml');
 
-        if (false !== $config['options']['legacy_twig_text_extension']) {
+        if (isset($bundles['SonataCoreBundle']) && false !== $config['options']['legacy_twig_text_extension']) {
```

Comment:
@franmomu 
You don't have to:
https://github.com/sonata-project/SonataAdminBundle/pull/6132#issuecomment-642455452